recipes/riassigner/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 build:
   noarch: python
   preserve_egg_dir: True
-  number: 0
+  number: 2
```

Comment:
Why is the build number 1 in the channel but not on github?

Reply:
I honestly don't know - we somehow got a mess into this with version and build numbers ... we were on 0.3.3 build number 1 and now want to go to 0.3.4 with changes in the meta.yaml - I tried build numbers 0, 1 and now 2, but so far unsuccessful!